Library/Homebrew/cli/named_args.rb

```diff
@@ -112,6 +115,17 @@ def to_formulae_and_casks_and_unavailable(only: parent&.only_formula_or_cask, me
       end
 
       def load_formula_or_cask(name, only: nil, method: nil, warn: nil)
+        if @without_api
+          Homebrew.without_api do
+            _load_formula_or_cask(name, only: only, method: method, warn: warn)
+          end
+        else
+          _load_formula_or_cask(name, only: only, method: method, warn: warn)
+        end
+      end
+      private :load_formula_or_cask
+
+      def _load_formula_or_cask(name, only: nil, method: nil, warn: nil)
```

Comment:
Not huge on this repeated naming, can one of them get a rename to something with `without_api` or something?

Reply:
Given they are both private anyway (which I didn't notice the first time): what about changing `load_formula_or_cask` to `load_formula_or_cask_without_api_if_needed`?